dashboard/test/ui/step_definitions/eyes_steps.rb

```diff
@@ -6,6 +6,10 @@
 
 When(/^I open my eyes to test "([^"]*)"$/) do |test_name|
   ensure_eyes_available
+
+  batch_name = test_name + " | " + ENV['BATCH_NAME']
+  @eyes.batch = Applitools::Base::BatchInfo.new(batch_name)
```

Comment:
My first attempt had us using a single batch per instance of eyes. It still resulted in a single top level item per test (i.e. it did not show multiple items like in the screenshot you have). I was not using the started_at param of BatchInfo.new, which maybe makes a difference. I can explore this a little further.